doc/source/cluster/kubernetes/user-guides/observability.md

```diff
@@ -0,0 +1,78 @@
+(kuberay-observability)=
+
+# KubeRay Observability
+
+## Ray Dashboard
+
+* To view the Ray dashboard running on the head Pod, users can follow [these instructions](kuberay-port-forward-dashboard).
```

Comment:
Would be nice to link to the main Ray Dashboard doc the first time we mention it.

Reply:
Update: https://github.com/ray-project/ray/pull/39074/commits/7c39cd2d2e8b9f983995c573188a0582a94ceab9